back/app/serializers/web_api/v1/folder_mini_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class WebApi::V1::FolderMiniSerializer < WebApi::V1::BaseSerializer
+  attributes :title_multiloc
+
+  attribute :visible_projects_count do |object, params|
+    params.dig(:visible_children_count_by_parent_id, object.admin_publication.id) || Pundit.policy_scope(user_context(params), Project).where(id: object.admin_publication.children.map(&:publication_id)).count
```

Comment:
I'm not totally clear what the `||` part is doing but it might be more efficient to do this in the controller if you can - think this will make an extra query per folder

Reply:
Just copied this over from the `folder_serializer`. `visible_children_count_by_parent_id` should always be available so I can remove it indeed